controllers/storagecluster/generate.go

```diff
@@ -45,10 +45,25 @@ func generateNameForCephBlockPool(initData *ocsv1.StorageCluster) string {
 }
 
 func generateDeviceClass(initData *ocsv1.StorageCluster) string {
```

Comment:
How about adding a unit test to cover this new logic?

Reply:
Although nice but as this function involves runtime updates in the cephcluster status it would be a lot of error to write a unit test for this, as it's a small bug fix I would not prefer spending a considerable amount of effort just for adding the unit test for this.

Reply:
Ok, I assumed it would be a small unit test for this one method.

Reply:
No, it actually depends on the populating of cephcluster status field with the deviceclasses.